datalad/support/annexrepo.py

```diff
@@ -184,6 +204,17 @@ def __init__(self, path, url=None, runner=None,
 
         self._batched = BatchedAnnexes(batch_size=batch_size)
 
+    def git_remote_add(self, name, url, options=''):
+        """Overrides method from GitRepo in order to set
+        remote.<name>.annex-ssh-options in case of a SSH remote."""
+        super(AnnexRepo, self).git_remote_add(name, url, options)
+        if url.startswith('ssh:'):
+            c = ssh_manager.get_connection(url)
+            self.repo.config_writer().set_value("remote \"%s\"" % name,
```

Comment:
stylistically imho it is better to use more of real estate of next lines, but starting all arguments in the following line, and not crumbling all up in a thin column on the right (I just see it here and below)